improver_tests/utilities/test_save.py

```diff
@@ -228,6 +228,23 @@ def test_remove_least_significant_digit(self):
         cube = load_cube(self.filepath)
         self.assertNotIn("least_significant_digit", cube.attributes)
 
+    def test_fill_value_no_mask(self):
+        """Test that fill_value is not set if there is no masked data."""
```

Comment:
This looks to me as if it is testing that the fill value IS set if no fill value is supplied. I'm not sure if the doc-string or the test needs to change. Possibly another unit test is needed?

Reply:
I probably didn't explain it well in the docstring, I meant that the fill value remains as a default value so isn't overriden. I've updated the docstring to explain this better